pkg/catalog/catalog.go

```diff
@@ -25,8 +25,19 @@ func New() *Catalog {
 	}
 }
 
-// Update updates the catalog by replacing existing profiles with new profiles
-func (c *Catalog) Update(sourceName string, profiles ...profilesv1.ProfileCatalogEntry) {
+// Append appends the existing profiles with new profiles
```

Comment:
-appends. The verb is already Append. :) I think that reads nicely.
// Append the existing profiles to the new profiles.